example-ideal.txt

```diff
@@ -0,0 +1,20 @@
+// Implementation example for empty picobot maze
+
+state leftTraverse {
+	move left;
+	else stay, downTraverse
```

Comment:
Seems like it would be nice for the `stay` to be implicit so users could just say `else downTraverse`.